.eslintrc.json

```diff
@@ -0,0 +1,333 @@
+        "computed-property-spacing": [
+            2,
+            "never"
+        ],
+        "eol-last": 2,
+        "func-names": 1,
+        "func-style": [
+            2,
+            "declaration",
+            {
+                "allowArrowFunctions": true
+            }
+        ],
+        "indent": [
+            2,
+            4,
+            {
+                "SwitchCase": 1
+            }
+        ],
+        "linebreak-style": 0,
+        "max-len": [
+            2,
+            200,
+            4,
+            {
+                "ignoreUrls": true,
+                "ignoreComments": false,
+                "ignorePattern": "^\\s*(const|let|var)\\s+\\w+\\s+\\=\\s+\\/.*\\/(|i|g|m|ig|im|gm|igm);?$"
+            }
+        ],
+        "max-nested-callbacks": [
+            2,
+            4
+        ],
+        "new-parens": 2,
+        "no-array-constructor": 2,
+        "no-lonely-if": 2,
+        "no-mixed-spaces-and-tabs": 2,
+        "no-multiple-empty-lines": [
+            2,
+            {
+                "max": 2,
+                "maxEOF": 1
+            }
+        ],
+        "no-nested-ternary": 2,
+        "no-new-object": 2,
+        "no-spaced-func": 2,
+        "no-trailing-spaces": 2,
+        "no-unneeded-ternary": 2,
+        "object-curly-spacing": [
+            2,
+            "always"
+        ],
+        "one-var": [
+            2,
+            "never"
+        ],
+        "padded-blocks": [
+            2,
+            "never"
+        ],
+        "quotes": [
+            1,
+            "single",
+            "avoid-escape"
+        ],
+        "semi-spacing": [
+            2,
+            {
+                "before": false,
+                "after": true
+            }
+        ],
+        "semi": [
+            2,
+            "always"
+        ],
+        "keyword-spacing": 2,
+        "space-before-blocks": 2,
+        "space-before-function-paren": [
+            2,
+            {
+                "anonymous": "always",
+                "named": "never"
+            }
+        ],
+        "space-in-parens": [
+            2,
+            "never"
+        ],
+        "space-infix-ops": 2,
+        "space-unary-ops": [
+            2,
+            {
+                "words": true,
+                "nonwords": false
+            }
+        ],
+        "spaced-comment": [
+            2,
+            "always",
+            {
+                "exceptions": [
+                    "-",
+                    "+"
+                ],
+                "markers": [
+                    "=",
+                    "!"
+                ]
+            }
+        ],
+
+        // Legacy
+        "max-depth": [
+            0,
+            4
+        ],
+        "max-params": [
+            2,
+            7
+        ],
+        "no-bitwise": 2
+    },
+    "globals": {
+        "$": true,
+        "ga": true,
+        "__ENV__": true,
+        "__DEVTOOLS__": true,
+        "PUBLIC_URL": true
+    }
+}
```

Comment:
It seems easier to extend some existing config rather than describe all rules manually.